Makefile

```diff
@@ -1,4 +1,3 @@
-REPO=integreatly
```

Comment:
+1 on removing this. We will never have to push this up manually to `integreatly` org. Makes sense for travis to just specify the repo rather than us pushing to this repo accidentally